Jint/Native/Function/FunctionInstance.cs

```diff
@@ -307,7 +307,7 @@ internal void OrdinaryCallBindThis(ExecutionContext calleeContext, JsValue thisA
             }
             else
             {
-                if (thisArgument.IsNullOrUndefined())
+                if (thisArgument == null || thisArgument.IsNullOrUndefined())
```

Comment:
Added this for convenience of calling a function from C# side with `null` thisArgument. Maybe this check can be moved inside `IsNullOrUndefined`, but it is used in a lot of places so I did not touch it.

Reply:
Nit: should use `is null` and `is not null` syntax against JsValues as equality goes to operator overloading unnecessarily.